gems/aws-sdk-sqs/CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 Unreleased Changes
 ------------------
 
+* Issue - Update QueuePoller to handle duplicate messages so that these can be deleted successfully.
```

Comment:
I think we may want to consider this a feature and minor version bump.  Also reference the issue it fixes like `(#2913)`.

Reply:
Either is fine IMO. Arguably it's a bug.